src/Service/auth.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import { API_PATH } from './constants';
+
+export const authAPI = {
+  login: (data) => axios.post(`${API_PATH}/login`, data),
+  register: (data) => axios.post(`${API_PATH}/register`, data),
+  getInfo: (data) => {
+    return axios.get(`${API_PATH}/users/${data.id}`);
+  },
```

Comment:
```suggestion
  getInfo: (data) => axios.get(`${API_PATH}/users/${data.id}`),
```
cho giống như 2 cái trên luôn nhé